webapp/src/test/java/org/apache/falcon/resource/EntityManagerJerseyIT.java

```diff
@@ -813,4 +813,38 @@ public Date getEndTime() {
         cal.set(Calendar.MILLISECOND, 0);
         return cal.getTime();
     }
+
+    @Test
+    public void testMultipleUpdateCommands() throws Exception {
```

Comment:
I am sending 2 update requests. So this will result in, 2 requests in separate threads going to the server.